CHANGELOG.md

```diff
@@ -8,6 +8,111 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Discovery Service (BPN Finder)](https://github.com/eclipse-tractusx/sldt-bpn-discovery)                                      |                                             bpndiscovery: [0.2.6](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/bpndiscovery-0.2.6)                                              |                                                                                                                                                          [0.3.1](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/v0.3.1)                                                                                                                                                           |
+| [Eclipse Data Space Connector (Tractus-X EDC)](https://github.com/eclipse-tractusx/tractusx-edc)                              |                                                    tractusx-connector: [0.7.1](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.1)                                                    |                                                                                                                                                              [0.7.1](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.1)                                                                                                                                                              |
+| [Eco Pass KIT](https://eclipse-tractusx.github.io/docs-kits/category/eco-pass-kit)                                            |                                                                                                  n/a                                                                                                   |                                                                                                                                                           [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Eco_Pass_KIT/changelog)                                                                                                                                                            |
+| [Golden Record Business Partner Number (BPN) Service](https://github.com/eclipse-tractusx/bpdm)                               |                                                            bpdm: [5.0.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-5.0.0)                                                             |                                                                                                                                                                 [v6.0.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/v6.0.0)                                                                                                                                                                 |
```

Comment:
add new table entry BEFORE:
| [Environmental and Social Standards (ESS) KIT](https://eclipse-tractusx.github.io/docs-kits/category/ess-kit)                 |                                        n/a                                         |                               [0.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/ESS-Kit/ESS%20Kit%20Changelog)